test/test_verify.py

```diff
@@ -275,6 +295,9 @@ def cleanTableStagePipe(self, connectorName, topicName="", partitionNumber=1):
             print(datetime.now().strftime("%H:%M:%S "), "=== Drop pipe {} ===".format(pipeName))
             self.snowflake_conn.cursor().execute("DROP pipe IF EXISTS {}".format(pipeName))
 
+        print(datetime.now().strftime("%H:%M:%S "), f"=== Delete Kafka topic {topicName} ===")
+        self.deleteTopic(topicName)
```

Comment:
Why exactly do we need to delete the topic here? Is this a fix, as it was missing, or did you add some logic related to that?